src/Workspaces/CoreTest/Remote/ServiceDescriptorTests.cs

```diff
@@ -98,8 +99,11 @@ void AddTypeRecursive(Type type, MemberInfo declaringMember)
 
                     foreach (var type in method.GetParameters().Select(p => p.ParameterType))
                     {
-                        // stream is special cased by JSON-RPC for streaming APIs
-                        if (type != typeof(Stream))
+                        // types that are special cased by JSON-RPC for streaming APIs
+                        if (type != typeof(Stream) &&
```

Comment:
no need to change.  however, teh direct type comparison seems suspect.  i.e. if someone took in a sublcass of any of these, this wouldn't work.  seems odd.

Reply:
I don't think subclass would work since these types are not serializable. They are special types handled by StreamJsonRpc. Not sure if they handle subtypes.
In any case we don't need subtypes and this is only checking for types we use.